#!/usr/bin/env python
PACKAGE = "example"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hip_y", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)
gen.add("hip_r", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)
gen.add("hip_p", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)
gen.add("knee", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)
gen.add("ank_p", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)
gen.add("ank_r", double_t, 0, "A double parameter",  1.00, -10.00,   10.00)


exit(gen.generate(PACKAGE, "example", "Convert"))